### ISSUE TITLE
Issue while installing both server and client on same machine

### ISSUE THREAD

**Issue by JoonasPit** [jisosomppi/log-analysis#4]
# Tried to install the both the server and client on the same machine.
 - Salt-master drops the other minion due to not being able to run two minions under the same ip.
# Solution proposition: 
 - Instruct users not to run scripts on same machine. 

- Issue also solved by simply being smarter

**Comment by OtsoR** [comment]
There could be a mention that serversetup.sh installs also its own minion: srv01 this s the minion that does not work when installing scripts on the same machine.

**Comment by elisapa** [comment]
Using Ubuntu 16.04 for master installation and Ubuntu 18.04 for minion.
When master and minions installed on different machines, everything works as intended in our test environment.

**Comment by jisosomppi** [comment]
Our structure relies on minion ID's to differentiate between different system types, hence the `srv*` ID on the master, and forced `ws-*` ID on all client systems. We stopped collecting log data from the server with an earlier version, after a logging loop created 450Gb+ of log data in less than an hour. We will return to this issue, should we have time.

For the time being, instructions have been updated to clarify the need of two systems.
